src/Logikfabrik.Umbraco.Jet/Logikfabrik.Umbraco.Jet.nuspec

```diff
@@ -15,7 +15,7 @@
     <copyright>Copyright (c) 2016</copyright>
     <tags>Umbraco, Jet, MVC, Code First, uJet</tags>
     <dependencies>
-      <dependency id="UmbracoCms.Core" version="[7.5.6,8.0]" />
+      <dependency id="UmbracoCms.Core" version="[7.6.0,8.0]" />
```

Comment:
In that case I think we should downgrade the project references, so that we compile with the same Umbraco version as referenced in the NuGet spec.